<?xml version="1.0" encoding="utf-8"?>

<launch>

    <!-- # arg -->
    <arg name="topic_imu" default="/imu" />
    <arg name="service_set_pose_odom" default="" />
    <arg name="service_set_pose_map" default="" />
    <arg name="frame_id_odom" default="" />
    <arg name="frame_id_map" default="" />

    <!-- # init_state -->
    <node pkg="ros_my_localization" type="init_yaw.py" name="init_yaw" clear_params="true" output="screen">
        <param name="topic_imu" value="$(arg topic_imu)" />
        <param name="service_set_pose_odom" value="$(arg service_set_pose_odom)" />
        <param name="service_set_pose_map" value="$(arg service_set_pose_map)" />
        <param name="frame_id_odom" value="$(arg frame_id_odom)" />
        <param name="frame_id_map" value="$(arg frame_id_map)" />
    </node>

</launch>
